kahuna/public/js/leases/leases.html

```diff
@@ -0,0 +1,83 @@
+<dt class="metadata-line__key flex-container">
+  <span class="flex-spacer">Leases</span>
+  <button ng:class="{'small': ctrl.grSmall}"
+          ng:click="ctrl.editing = true"
+          gr:tooltip="Add lease"
+          gr:tooltip-position="left">
+
+      <gr-icon ng-show="!ctrl.editing" ng:class="{'spin': ctrl.adding}">add_box</gr-icon>
+      <span>
+         <span ng:show="ctrl.adding">Adding...</span>
+      </span>
+  </button>
+</dt>
+
+<form class="lease__form"
```

Comment:
http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/ Is a good one.